### ISSUE TITLE
Crimson Dance Issue

### ISSUE THREAD

**Issue by B14d3r11** [Openarl/PathOfBuilding#1088]
The "Bleed can stack up to 8 times" isn't functioning so I am not getting a proper estimate of 8 stacks of bleed on an enemy...

**Comment by Openarl** [comment]
It isn't functioning because it isn't supposed to. I haven't added support for it yet, as it is very difficult to simulate correctly.

**Comment by B14d3r11** [comment]
Didn't Crimson Dance just make it similar to poison though with a max stack of 8?

**Comment by Openarl** [comment]
It's the "max stack" part that complicates matters, particularly the fact that it's the largest 8 stacks that deal damage at any one point in time.

**Comment by B14d3r11** [comment]
So if I have 8k bleed dps, 8 stacks would mean 64k bleed dps if all 8 stacks are on the person at the same given moment right?

**Comment by Openarl** [comment]
Only if your Bleed stacks are always roughly the same DPS, which is only true if you're non-crit, and not using Ruthless or any source of Double Damage.

**Comment by B14d3r11** [comment]
Yeah, i don't have crits and no double damage. So thanks, that atleast helps me figure out what my dps is with Crimson Dance now :D
